Client/DeviceCheck/Cryptography.swift

```diff
@@ -0,0 +1,240 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+/// An error class representing an error that has occurred when handling encryption
+public struct CryptographyError: Error {
+  //The error domain
+  public let domain: String
+  
+  //The error code
+  public let code: Int32
+  
+  //A description of the error
+  public let description: String?
+  
+  init(code: Int32, description: String? = nil) {
+    self.domain = "com.brave.security.cryptography.error"
+    self.code = code
+    self.description = description
+  }
+}
+
+/// A class representing a cryptographic key.
+public struct CryptographicKey {
+  private let key: SecKey
+  private let keyId: String
+  
+  public init(key: SecKey, keyId: String) {
+    self.key = key
+    self.keyId = keyId
+  }
+  
+  /// Returns the private key
+  public func getPrivateKey() -> SecKey {
```

Comment:
Do you expect the private key to be used outside this file?

Reply:
True. I will fix it. The private key is not needed outside this file but also, the private key can't be exported as well so it doesn't hurt :)

But yeah, good point. It will be fixed.